calm_adapter/calm_deletion_checker/src/main/scala/uk/ac/wellcome/platform/calm_deletion_checker/Main.scala

```diff
@@ -0,0 +1,12 @@
+package uk.ac.wellcome.platform.calm_deletion_checker
+
+import akka.Done
+import uk.ac.wellcome.typesafe.WellcomeTypesafeApp
+
+import scala.concurrent.Future
+
+object Main extends WellcomeTypesafeApp {
+  runWithConfig { _ => () =>
+    Future.successful(Done)
```

Comment:
💯 for advanced functionality

Reply:
This bit is final, just to make sure I don't introduce any bugs